src/components/vanilla/charts/StackedBarChart/index.tsx

```diff
@@ -56,11 +63,33 @@ export default (props: Props) => {
       : props.results,
   };
 
+  if (props.showTotals) {
+    const totals: Totals = {};
+    const { data } = props.results;
+    const { metric, xAxis, segment } = props;
+    if (data && data.length > 0) {
+      data?.forEach((d: { [key: string]: any }) => {
+        const x = d[xAxis.name];
+        const y = parseInt(d[metric.name], 10);
```

Comment:
@cwbuecheler should this be parseFloat? I've used parseInt in the past in various places and then realised it doesn't show decimals where it should.